packages/node/src/handlers.ts

```diff
@@ -244,10 +245,10 @@ export async function verifyIncomingVertices(
 	});
 
 	const drp = object.drp as DRP;
-	if (!drp.acl) {
+	const acl: IACL & DRP = object.acl as IACL & DRP;
```

Comment:
```suggestion
	const acl: IACL = object.acl as IACL;
```
You don't need the `DRP` interface and why are you casting the type?

Reply:
ignore the part of "why are u casting the type"